config/locales/en.yml

```diff
@@ -674,12 +674,16 @@ en:
         reason: Cancellation reasons
         rejection_category:
           fully_booked: The date you requested is fully booked
-          accepted_on_ttc: We cannot offer you school experience because you have already been accepted on a Teacher Training Course
-          date_not_available: We cannot support the date you have requested
-          no_relevant_degree: We do not believe you have a relevant degree for the school experience you are applying for
-          no_phase_availability: We are unable to offer you school experience for the teaching phase you are interested in
-          candidate_not_local: We are looking for candidates that live locally to the school
+          accepted_on_ttc: We cannot offer you school experience because you've already been accepted on a teacher training course
```

Comment:
This is fine, just needs a full stop at the end so that the sentence looks complete when its rendered in the candidate email.